index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>my first microverse propject</title>
+    
```

Comment:
- [OPTIONAL] I think it would be a good idea to remove unnecessary blank lines from your code, in order to maintain your code clean and readable :wink: